worker/app/startup_tests.py

```
@@ -158,16 +158,48 @@ def test_decoder(decoder_name: str, hw_flags: List[str]) -> Tuple[bool, str]:
+                "-f", "null", "-",
+            ]
+        else:
+            # NVENC / CPU: simple test (original logic)
+            cmd = [
+                "ffmpeg", "-hide_banner",
+                "-f", "lavfi", "-i", "color=black:s=256x256:d=0.1",
+                "-c:v", encoder_name, "-t", "0.1", "-frames:v", "3",
+                "-f", "null", "-",
+            ]
+
         result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)